src/gt4py/next/program_processors/runners/dace_fieldview/gtir_builtin_translators.py

```diff
@@ -188,18 +234,44 @@ def translate_as_field_op(
     assert isinstance(domain_expr, gtir.FunCall)
 
     # parse the domain of the field operator
-    domain = dace_gtir_utils.get_domain(domain_expr)
+    domain = translate_domain(domain_expr)
 
+    # The reduce identity value is used to fill the skip values in neighbors list
+    #
+    # A reduction expression can be either expressed in local view (itir):
+    # vertices @ u⟨ Vertexₕ: [0, nvertices) ⟩
+    #      ← as_fieldop(
+    #          λ(it) → reduce(plus, 0)(neighbors(V2Eₒ, it)), u⟨ Vertexₕ: [0, nvertices) ⟩
+    #        )(edges);
+    #
+    # or in field view (gtir):
+    # vertices @ u⟨ Vertexₕ: [0, nvertices) ⟩
+    #      ← as_fieldop(λ(it) → reduce(plus, 0)(·it), u⟨ Vertexₕ: [0, nvertices) ⟩)(
+    #          as_fieldop(λ(it) → neighbors(V2Eₒ, it), u⟨ Vertexₕ: [0, nvertices) ⟩)(edges)
+    #        );
+    #
+    # In local view, the list of neighbors is (recursively) built while visiting
+    # the current expression.
+    # In field view, the list of neighbors is built as argument to the current
+    # expression. Therefore, the reduction identity value needs to be passed to
+    # the argument visitor (`reduce_identity_for_args = reduce_identity`).
+    # However, when the argument visitor hits the `neighbors` expression (see the
+    # second if-branch), we should stop carrying the reduce identity further
+    # (`reduce_identity_for_args = None`).
```

Comment:
Okay, but why?

Reply:
The reason why we stop carrying the reduce identity further is that we do not need it, since reduce operates along a single axis. I have updated the comment.